custom/icds_reports/utils/aggregation_helpers/distributed/bihar_api_demographics.py

```diff
@@ -132,7 +133,10 @@ def indexes(self):
         return [
             f"""CREATE INDEX IF NOT EXISTS demographics_state_person_case_idx
                 ON "{self.monthly_tablename}" (month, state_id, person_id)
-            """
+            """,
+            f"""CREATE INDEX IF NOT EXISTS demographics_gender_dobx
```

Comment:
good to have this index on the parent table instead of here. You add it to the migration file. 
@calellowitz do you think this index anymore columns needs to be added along with gender and dob?

Reply:
Based on my read of the queries (assuming its just those two), you want (gender, dob, person) or (state, gender, dob, person). month wont be necessary because the child table is already restricted to a single month

Reply:
Thanks @calellowitz ,
@challabeehyv could you create an index (gender, dob, person) in the migration file. I state is fine not to include as this only has one state data.